SwiftLeeds/Network/ScheduleEndpoint.swift

```diff
@@ -7,11 +7,6 @@
 
 import NetworkKit
 
-// MARK: - Environment
```

Comment:
This should be put back to `Endpoints.swift`, named on purpose to contain all endpoints of the App (as there aren't many and saves lots of small files)